controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentContext.java

```diff
@@ -529,11 +519,6 @@ private Run currentRun(JobId job) {
         return run;
     }
 
-    /** Sets a single endpoint in the routing layer for the tester instance in this */
-    private DeploymentContext setTesterEndpoints(ZoneId zone) {
-        return setEndpoints(zone, true);
-    }
-
     /** Sets a single endpoint in the routing layer; this matches that required for the tester */
     private DeploymentContext setEndpoints(ZoneId zone, boolean tester) {
```

Comment:
If the tester endpoints are indeed gone, the `tester` parameter  should be removed and the javadoc updated.

Reply:
Yup, fixed